monkey/tests/unit_tests/monkey_island/cc/resources/test_island_mode.py

```diff
@@ -54,11 +53,7 @@ def test_island_mode_post__invalid_mode(flask_client):
 
 
 def test_island_mode_post__internal_server_error(build_flask_client):
```

Comment:
Why does this test still pass? What is raising an internal server error?

Reply:
No instances of ISimulationRepository and IIslandEventQueue are registered with the DI container for this test.